source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/OverflowActionLayoutRenderer.java

```diff
@@ -0,0 +1,361 @@
+    {
+        BottomSheetDialog bottomSheetDialog = new BottomSheetDialog(context, R.style.BottomSheetDialogTheme);
+        LinearLayout contentLayout = new LinearLayout(context);
+        contentLayout.setLayoutParams(new LinearLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT));
+        contentLayout.setOrientation(LinearLayout.VERTICAL);
+        contentLayout.setGravity(Gravity.START);
+        addSeparator(contentLayout);
+        renderSecondaryActionElements(baseActionElementList, renderedCard, context, fragmentManager, viewGroup, contentLayout, cardActionHandler, hostConfig, renderArgs);
+        bottomSheetDialog.setContentView(contentLayout);
+
+        return bottomSheetDialog;
+    }
+
+    /**
+     * Applies backgrounds drawables to the Overflow view.
+     *
+     * @param context context
+     * @param view    drawables to be applied on view.
+     */
+    private void applyBackgroundDrawables(Context context, @NonNull ImageButton view)
+    {
+        TypedValue colorValue = new TypedValue();
+        //Apply colorButtonNormal for border tp match with existing action buttons;
+        int color = context.getResources().getColor(android.R.color.darker_gray);
+        if (context.getTheme().resolveAttribute(android.R.attr.colorButtonNormal, colorValue, true))
+        {
+            color = getAttributeColor(context, R.attr.colorButtonNormal);
+        }
+        else if (context.getTheme().resolveAttribute(android.R.attr.colorPrimary, colorValue, true))
+        {
+            color = getAttributeColor(context, R.attr.colorPrimary);
+        }
+
+        ShapeDrawable shapedrawable = getShapeDrawable(context, color, 2);
+        view.setColorFilter(color, PorterDuff.Mode.SRC_ATOP);
+        TypedValue outValue = new TypedValue();
+        context.getTheme().resolveAttribute(android.R.attr.selectableItemBackground, outValue, true);
+        Drawable backGroundDrawable = ContextCompat.getDrawable(context, outValue.resourceId);
+        Drawable[] layers = {backGroundDrawable, shapedrawable};
+        LayerDrawable layerDrawable = new LayerDrawable(layers);
+        view.setBackground(layerDrawable);
+    }
+
+    /**
+     * Adds divider to the BottomSheet's content layout.
+     *
+     * @param linearLayout BottomSheet's root layout
+     */
+    private void addSeparator(@NonNull LinearLayout linearLayout)
+    {
+        ShapeDrawable shapedrawable = getShapeDrawable(linearLayout.getContext(), linearLayout.getContext().getResources().getColor(R.color.bottomSheet_divider_color), 1);
+        linearLayout.setDividerDrawable(shapedrawable);
+        linearLayout.setShowDividers(LinearLayout.SHOW_DIVIDER_MIDDLE);
+    }
+
+    /**
+     * Applies border with transparent background to the Overflow View.
+     *
+     * @param context     context to be used for rendering
+     * @param color       color of the border
+     * @param strokeWidth width of tjhe border
```

Comment:
typo

```suggestion
     * @param strokeWidth width of the border
```

Reply:
missed in the last commit, but will push it with next commit.